packages/core/src/agent/Agent.ts

```diff
@@ -70,19 +71,17 @@ export class Agent {
 
     // Based on interfaces. Need to register which class to use
     this.container.registerInstance(InjectionSymbols.Logger, this.logger)
-    this.container.registerInstance(InjectionSymbols.Indy, this.agentConfig.indy)
     this.container.register(InjectionSymbols.Wallet, { useToken: IndyWallet })
     this.container.registerSingleton(InjectionSymbols.StorageService, IndyStorageService)
     this.container.registerSingleton(InjectionSymbols.MessageRepository, InMemoryMessageRepository)
 
-    // File system differs based on NodeJS / React Native
-    this.container.registerInstance(InjectionSymbols.FileSystem, this.agentConfig.fileSystem)
+    // Platform specific dependencies
+    this.container.registerInstance(InjectionSymbols.FileSystem, new dependencies.FileSystem())
```

Comment:
Nitpick: What about creating an instance outside? Then you will have more flexibility to pass params into the constructor or set the instance as you need before passing it into the agent

Reply:
I'll look at that in a separate PR. That would mean extra steps to get started, even though you won't need in 99% of the cases

We can make it an optional config property?